---
layout: with-nav
---

# Berlin RNA toolbox
deNBI RNA Bioinformatics Centre Berlin node provides several tools for RNA bioinformatics. The tools provided are listed below. 


### Databases
[doRiNA](http://dorina.mdc-berlin.de): a database of RNA interactions in post-transcriptional
regulation

[circBase](http://circbase.mdc-berlin.de): a database for public circular RNA datasets and tools to discover
circRNAs in RNA-seq data; PicTar: a microRNA target predictor; miRDeep: a probabilistic
model that detects the presence of expressed animal microRNAs in deep sequencing data;

### Tools
[PicTar](https://pictar.mdc-berlin.de/): a microRNA target predictor; 

[miRDeep](https://www.mdc-berlin.de/content/mirdeep2-documentation): a probabilistic model that detects the presence of expressed animal microRNAs in deep sequencing data;

[PIPmiR](https://ohlerlab.mdc-berlin.de/software/Pipeline_for_the_Identification_of_Plant_miRNAs_84/): a tool to identify novel plant miRNA genes from a combination of deep sequencing and
genomic features; 

[NASTIseq](https://ohlerlab.mdc-berlin.de/software/NASTIseq_104/): expression analysis for the identification of cis-Natural Antisense Transcripts (cis-NAT) from strand specific RNA-seq data; 

[PARalyzer](https://ohlerlab.mdc-berlin.de/software/PARalyzer_85/): a peak finder for protein-
RNA interaction sites in PAR-CLIP data; microMUMMIE: microRNA target-site prediction in PAR-CLIP data

[cERMIT](https://ohlerlab.mdc-berlin.de/software/cERMIT_82/): a motif finder for large sequence sets e.g. from chromatin or RNA immunopreciptation experiments.



